customize_image.sh

```diff
@@ -60,6 +65,25 @@ cleanup() {
 
 trap cleanup ERR SIGINT
 
+# resize partition 2 to END=3964MB if necessary
```

Comment:
This comment just tells what the code does, but not why.

Reply:
I've amended the last commit , I'm ready for feedback ;-)